vignettes/converting-the-data-for-faster-analysis.qmd

```diff
@@ -18,9 +18,9 @@ execute:
 
 `spod_get()` returns objects that are only appropriate for small datasets representing a few days of the national OD datasets. We recommend converting the data into analysis-ready formats (`DuckDB` or `Parquet`), allowing you to work with large datasets on a consumer laptop (with 8-16 GB of memory).
 
-**Converting the data to either `DuckDB` or `Parquet` will improve the computation time up to 100x compared to continuing working with the tables that you get from `spod_get()` function that reads CSV files every time you do anything.**
+**Converting the data to either `DuckDB` or `Parquet` will improve the computation time up to 100x compared with using data frames returned with `spod_get()`, which reads CSV files every time you call it.**
```

Comment:
Ok, to be fair, it is more like up to 8-10x, not 100x speed improvement of DuckDB database over CSV files.
![Screenshot 2024-08-26 at 13 31 48](https://github.com/user-attachments/assets/4d4109dc-6090-4e6c-ad88-b42ff99f31b9)